### PR TITLE
DHFPROD-6058: Update setup.sh script for DHS

### PR DESCRIPTION
**Pull request by briantang**
### Description

- Updated setup.sh script with two parameters for deploying project on DHS

#### Checklist: 
```diff
- Note: do not change the below
```

-  ##### Owner:

- [x] JIRA_ID included in all the commit messages
- [x] PR title is in the format JIRA_ID:Title
- [x] Rebase the branch with upstream
- [x] Squashed all commits into a single commit
- [x] Added Tests
  

- ##### Reviewer:

- [x] Reviewed Tests

### CODE REVIEW

File: marklogic-data-hub-central/ui/e2e/setup.sh [hunk 1]
```diff
@@ -20,6 +22,7 @@ if $DHS
 then
         env=dhs
         perl -i -pe"s/mlHost=.*/mlHost=$mlHost/g" gradle-dhs.properties
+        ./gradlew -i mldeployusers -PenvironmentName=$env -PmlUsername=$mlAdminUsername -PmlPassword=$mlAdminPassword
         ./gradlew hubDeploy -PenvironmentName=$env --info --stacktrace
 else
         ./gradlew mlDeploy -PmlUsername=admin -PmlPassword=admin --info --stacktrace
```
Comment (ryanjdew) [inline marklogic-data-hub-central/ui/e2e/setup.sh:28]: If we are requiring mlAdminUsername and mlAdminPassword for both dhs and local, should we be using those variables here as well?

Comment (briantang) [reply marklogic-data-hub-central/ui/e2e/setup.sh:28]: Right, thanks @ryanjdew will make that change

**Review by SameeraPriyathamTadikonda — APPROVED** [review]

File: Jenkinsfile [hunk 2]
```diff
@@ -191,7 +191,7 @@ def runCypressE2e(){
           repo="maven {url '"$M2_LOCAL_REPO"'}"
           sed -i "/repositories {/a$repo" hc-qa-project/build.gradle
           chmod +x setup.sh;
-          ./setup.sh dhs=false mlHost=localhost;
+          ./setup.sh dhs=false mlHost=localhost mlAdminUsername=admin mlAdminPassword=admin;
```
Comment (rjrudin) [inline Jenkinsfile:194]: Use mlSecurityUsername/Password instead - see https://github.com/marklogic-community/ml-gradle/wiki/Property-reference#manage-server-connection-properties

**Review by rjrudin — CHANGES_REQUESTED** [review]

**Review by Sanjeevani19 — APPROVED** [review]

File: Jenkinsfile (hunk 2 above)
Comment (SameeraPriyathamTadikonda) [reply Jenkinsfile:194]: Can we use mlSecurityUsername on DHS too?

Comment (briantang) [reply Jenkinsfile]: Thanks @rjrudin, modified the parameter names

**Review by rjrudin — APPROVED** [review]

**Review by SameeraPriyathamTadikonda — APPROVED** [review]

**Review by Sanjeevani19 — APPROVED** [review]

**Pull request merged**